src/components/ScrollShadows.tsx

```diff
@@ -22,17 +22,24 @@ export function ScrollShadows(props: ScrollShadowsProps) {
     throw new Error("ScrollShadows: bgColor prop must be in the format 'rgba(255, 255, 255, 1)'");
   }
 
-  const transparentBgColor = bgColor.replace(/,1\)$/, ",0)");
-  const startShadowStyles = !horizontal ? Css.top0.left0.right0.hPx(40).$ : Css.left0.top0.bottom0.wPx(25).$;
-  const endShadowStyles = !horizontal ? Css.bottom0.left0.right0.hPx(40).$ : Css.right0.top0.bottom0.wPx(25).$;
-  const startGradientDeg = !horizontal ? 180 : 90;
-  const endGradientDeg = !horizontal ? 0 : 270;
-
   const [showStartShadow, setShowStartShadow] = useState(false);
   const [showEndShadow, setShowEndShadow] = useState(false);
-
   const scrollRef = useRef<HTMLDivElement>(null);
 
+  const [startShadowStyles, endShadowStyles] = useMemo(() => {
+    const transparentBgColor = bgColor.replace(/,1\)$/, ",0)");
+    const commonStyles = Css.absolute.z3.$;
```

Comment:
@stephenh - The zIndex (`z3`) is set here.

Reply:
Ack! I grepped for `zIndex` and forgot our shortcut. :speak_no_evil: :-D Thanks!